### ISSUE TITLE
new version tag / binaries with fixed labels ?

### ISSUE THREAD

**Issue by beorn-** [prometheus-community/elasticsearch_exporter#328]
Hi,

You fixed a lot of lables (for obvious good reasons) but you did not tag a new version/release. Could you do it please ?

**Comment by zwopir** [comment]
Hi Björn,

I try to do my best. Times are a bit crazy those days.

I already tried to address this topic in
https://github.com/justwatchcom/elasticsearch_exporter/issues/320? 
but haven't received a feedback yet. This topic is at least related

**Comment by beorn-** [comment]
I'm not THE Björn from the prometheus core team. I was just hoping for a tagged release with all the good label changes. If you need help just let me know. i might just do that

**Comment by zwopir** [comment]
yes, I saw the additional "-". Thanks for offering your help. I'll ping you if I need your help

**Comment by tiagomeireles** [comment]
I would also love to see a new release.
